include/hip/hip_hcc.h

```diff
@@ -27,6 +27,10 @@ THE SOFTWARE.
 
 #include "hip/hip_runtime_api.h"
 
+// HIP implicit kernargs.
+#define HIP_IMPLICIT_KERNARG_SIZE 48
+#define HIP_KERNARG_ALIGNMENT 8
```

Comment:
better not introduce this macro since it causes confusion. You align up the offset of the first implicit arg because it has alignment of 8 itself, not because there is generic alignment requirement for kernel args.